src/utilities/updateWindowLocation.spec.ts

```diff
@@ -0,0 +1,20 @@
+// @vitest-environment happy-dom
```

Comment:
worth calling out I'm targeting happy-dom as the environment for this test file specifically which allows testing window.location. The other tests are still running in the node environment which I think we want to keep because of ssr support which we will need to support.